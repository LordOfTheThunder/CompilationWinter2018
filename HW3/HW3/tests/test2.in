void TestBool (int x) {
	bool k = false;
}
void TestBoolMismatch(bool a) {
	int k = true;
}




void main() { print("I hate this course"); }



