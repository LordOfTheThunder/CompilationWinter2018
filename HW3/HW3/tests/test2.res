---end scope---
x INT -1
k BOOL 0
line 5: type mismatch
