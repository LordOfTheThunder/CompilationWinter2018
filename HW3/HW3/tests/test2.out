---end scope---
x INT 1
k BOOL 1
line 5: type mismatch
